gradle/init-scripts/build-scan.init.gradle.kts

```diff
@@ -29,6 +31,10 @@ fun configureExtension(extension: Any) {
         }
         setProperty("server", "https://e.grdev.net")
 
+        if (ci) {
+            setProperty("uploadInBackground", false)
```

Comment:
:o: Why is this required if this is already done in BuildScanPlugin.kt ?

Reply:
I wasn't sure how these were used that is why I added it on both places. Removed it.